cli/tests/integration/shared_recovery/delete.rs

```diff
@@ -0,0 +1,81 @@
+use libparsec::{tmp_path, TmpPath};
+
+use crate::testenv_utils::{TestOrganization, DEFAULT_DEVICE_PASSWORD};
+use crate::utils::*;
+
+use crate::integration_tests::bootstrap_cli_test;
+
+#[rstest::rstest]
+#[tokio::test]
+async fn remove_shared_recovery_ok(tmp_path: TmpPath) {
```

Comment:
Try to limit call to the cli, each call add 2sec (on my machine) to the test duration.

Here some call could be removed since they're tested elsewhere (I think of the 2 `info` command)

Reply:
I just noticed the cli tests do not use the testbed pre-configured organization. Those would solve the slowness you are mentioning. I suggest we:
- factor test initialization into a fixture (i.e. the creation of shamir recoveries) 
- create an issue to migrate the cli tests to the testbed pre-configured organization